compilers/concrete-compiler/compiler/lib/Bindings/Rust/build.rs

```diff
@@ -5,7 +5,22 @@ use std::error::Error;
 use std::path::Path;
 use std::process::exit;
 
-const MLIR_STATIC_LIBS: [&str; 174] = [
+const MLIR_STATIC_LIBS: [&str; 189] = [
```

Comment:
I think you can put a `_` here instead of 189

Reply:
the compiler complains about it, although I tested it with stable, is it only available in nightly?

Reply:
It's unstable indeed  
But changing (the four lists) to `: &[&str] = &[` works

Reply:
updated